extra/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
```

Comment:
Consider adding default values here `1000` and using that variable below, instead of using the value `1000` in so many places.

Reply:
You are absolutely right here: It works, but looks ugly and must be corrected. Sorry for this - there must be some kind of "blackout" after a lot of tests with different configs I made yesterday. I'll fix it.

Reply:
Corrected with 7f78cc8d0febeb01308a42e05aea6bfab8d67633